src/com/cft/contactmerge/contact/FirstNameProperty.java

```diff
@@ -0,0 +1,27 @@
+package com.cft.contactmerge.contact;
+
+import com.cft.contactmerge.AnswerType;
+
+public class FirstNameProperty implements IContactProperty<String> {
+
+    private String firstName;
+
+    public FirstNameProperty(String firstName)
+    {
+        this.firstName = firstName;
+    }
+
+    public AnswerType isMatch(String otherProperty) {
+        // TODO: Add code to return appropriate response
+        return null;
+    }
+
+    public String getValue()
+    {
```

Comment:
Not sure why we have two methods that do exactly the same thing. Is there a reason for this?

Reply:
I assume you mean getValue () and toString()? They just happen to return the same value in this case. See "NameProperty" for an example where they don't return the same thing.